### ISSUE TITLE
About Response Reward

### ISSUE THREAD

**Issue by huangfuyb** [qiancheng0/ToolRL#22]
Hi, Greate Job, Thanks for public. I have a question about reward.
The assistant is required for the response to user, such as clarifying or summarizing based on the tool results. That is, \<response\>xxx\</response\>. Is the only  reward for the formatting? The content  accuracy is not considered?

**Comment by qiancheng0** [comment]
First, the response field is not always required in our training setting (some ground truth does not contain a response field). The response reward is not considered in the training as we are mainly training the model's tool-calling ability, and all the training tasks require selecting the right function and parameters for tool-calling, instead of making correct responses.

**Comment by huangfuyb** [comment]
OK, Get it, thanks. And as this setting, the example of Table 4 of paper is  seen OK.
